common/lib/xmodule/xmodule/library_content_module.py

```diff
@@ -649,6 +649,19 @@ def has_dynamic_children(self):
         """
         return True
 
+    @property
+    def has_gated_content(self):
+        """
+        Tells whether library contains questions only accessible to upgraded learners
```

Comment:
Minor: "Returns whether the library contains any questions that are only accessible to upgraded/paid learners"